### PR TITLE
Eve Xml Kills

### PR DESCRIPTION
**Pull request by clonerworks**
Eve ship kills added

### CODE REVIEW

File: docs/xmlapi/endpoints.md [hunk 1]
```diff
@@ -112,6 +112,7 @@
 * [RefTypes](eve_reftypes.md)
 
 ## Map
+* [Kills](map_kills.md
```
Comment (Regner) [inline docs/xmlapi/endpoints.md]: This line seems to be missing a closing )

Comment (clonerworks) [inline docs/xmlapi/endpoints.md]: Fixed that issue. Thanks for finding it.

**Comment by clonerworks** [comment]
off topic but do we have a section for common sql or mysql queries for the SDE?

**Comment by Regner** [comment]
No but we should.

**Pull request merged**